path_login_test.go

```diff
@@ -53,10 +53,11 @@ func setupBackend(t *testing.T, noPEMs bool) (logical.Backend, logical.Storage)
 	}
 
 	req = &logical.Request{
-		Operation: logical.CreateOperation,
-		Path:      "role/plugin-test",
-		Storage:   storage,
-		Data:      data,
+		Operation:  logical.CreateOperation,
+		Path:       "role/plugin-test",
+		Storage:    storage,
+		Data:       data,
+		Connection: &logical.Connection{},
```

Comment:
Rather than adding empty Connection blocks in tests, you're better off checking for nil connection info in the actual login/renewal code, and failing if cidrs are defined but it's nil.

Reply:
Hm, good point. It's way less code too.